packages/ui/src/createModal.ts

```diff
@@ -78,6 +88,7 @@ export const createFullscreenModal = (
     modal.update({ loading: true });
 
     Promise.resolve(contentElement()).then((element) => {
+      modal.replaceChildren();
```

Comment:
Probably we can pass element to this function and remove next `appendChild` line?
```suggestion
      modal.replaceChildren(element);
```